lib/money/money.rb

```diff
@@ -259,6 +264,10 @@ def to_fs(style = nil)
   alias_method :to_s, :to_fs
   alias_method :to_formatted_s, :to_fs
 
+  def to_money_string
+    "#{currency.to_s} #{to_s(:amount)}"
```

Comment:
I believe the `to_s` on `currency` is redundant here.
```suggestion
    "#{currency} #{to_s(:amount)}"
```

Reply:
I think it's required:
```
irb(main):001> m = Money.new(56.78, 'SEK')
=> #<Money value:56.78 currency:SEK>
irb(main):002> m.currency
=>
#<Money::Currency:0x000000011ff32800
 @decimal_mark=",",
 @disambiguate_symbol="SEK",
 @iso_code="SEK",
 @iso_numeric="752",
 @minor_units=2,
 @name="Swedish Krona",
 @smallest_denomination=100,
 @subunit_symbol=nil,
 @subunit_to_unit=100,
 @symbol="kr">
irb(main):003> m.currency.to_s
=> "SEK"
```

Reply:
Since you are already inside a string you get it for free :smile:. I'm using a version this method until this is merged and Rubocop complained about the additional .to_s inside the interpolated string.

```
irb(main):002> m = Money.new(56.78, 'SEK')
=> #<Money value:56.78 currency:SEK>
irb(main):003> m.currency
=> #<Money::Currency:0x0000000100239118 @decimal_mark=",", @disambiguate_symbol="SEK", @iso_code="SEK", @iso_numeric="752", @minor_units=2, @name="Swedish Krona", @smallest_denomination=100, @subunit_symbol=nil, @subunit_to_unit=100, @symbol="kr">
irb(main):004> "#{m.currency}"
=> "SEK"
irb(main):005> "#{m.currency.to_s}"
=> "SEK"
```